src/+matlode/+rk/+erk/ERK.m

```diff
@@ -78,40 +50,51 @@
             tCur = tspan(1);
             tNext = tCur;
             ti = 2;
-            hmax = opts.MaxStep;
+            
+            tdir = sign(tspan(end) - tspan(1));
```

Comment:
Somewhere before this, do we check that tspan(end) ~= tspan(1) ? There is an edge case if tdir can potentially be zero.

Reply:
actually this brings up another weird edge case of tspan(i) == tspan(j) where i ~=j.     issorted(tspan, 'strictascend') is an easy way to check before passing